src/actions/actionExport.tsx

```diff
@@ -12,6 +12,7 @@ export const actionChangeProjectName: Action = {
   },
   PanelComponent: ({ appState, updateData }) => (
     <EditableText
+      label="File title"
```

Comment:
Shall we use a new section in the translations?

Reply:
Yes! I'm thinking of sending those in a follow-up. But I can do it in the current PR if that would be better.